descriptors/ModuleDescriptor-template.json

```diff
@@ -156,7 +157,8 @@
             "circulation-storage.loan-policies.item.get",
             "circulation-storage.request-policies.item.get",
             "circulation-storage.fixed-due-date-schedules.item.get",
-            "circulation-storage.fixed-due-date-schedules.collection.get"
+            "circulation-storage.fixed-due-date-schedules.collection.get",
+            "configuration.entries.collection.get"
```

Comment:
Please add a dependency defined for the `configuration` interface

Reply:
done